### DISCUSSION TITLE
Create a Way to Embed a Series of Posts

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16805] [Icebox]
I think it'd be awesome if Forem users were given the option to embed series!

This request was outlined thoroughly by @seangwright previously here - https://github.com/forem/forem/issues/3046. Just to point out that our architecture for embeds (previously referred to as Liquid Tags) has [changed](https://dev.to/devteam/feature-update-rich-content-embeds-390e) since this request was written.

We offer embeds for a large variety of internal types of content, so I think including series would be a really great move that is in line with our other options:

- Forem Community Comment
- Forem Community Link
- Forem Community Link
- Forem Community Listing
- Forem Community Organization
- Forem Community Podcast Episode
- Forem Community Tag
- Forem Community User Profile

Also, here's [an example of demand for this feature](https://dev.to/moopet/comment/1mgbi) on dev.to.

**Comment by JennieOcken** [comment]
There are a lot of requests around making our series functionality more robust.  I want to group them all together here and say we will take them as a set at some point in the future.

[Write description](https://github.com/forem/forem/discussions/15474)
[Edit](https://github.com/forem/forem/discussions/15673)
[Delete](https://github.com/forem/forem/discussions/16434)
[Embed](https://github.com/forem/forem/discussions/16805)
[Multiple Authors](https://github.com/forem/forem/discussions/15320)
[Reorder](https://github.com/forem/forem/discussions/15391)
